scss/action-buttons/_theme.scss

```diff
@@ -1,6 +1,7 @@
 @include exports('action-buttons/theme') {
 
     .k-action-buttons {
+        border-color: $popup-border;
```

Comment:
I think this is handled by https://github.com/telerik/kendo-theme-default/blob/develop/scss/action-buttons/_layout.scss#L6, so we don't really need it.

You could add a class name to `kendo-timeselector` and basically inherit border-color.